pkg/scalers/aws_cloudwatch_scaler.go

```diff
@@ -189,6 +190,12 @@ func parseAwsCloudwatchMetadata(config *scalersconfig.ScalerConfig) (*awsCloudwa
 	}
 	meta.minMetricValue = minMetricValue
 
+	ignoreNullValues, err := getParameterFromConfigV2(config, "ignoreNullValues", reflect.TypeOf(true), IsOptional(true), UseMetadata(true), WithDefaultVal(true))
```

Comment:
`minMetricValue` is still a required option above. I haven't changed that, as I wasn't sure how to handle the behaviour here. Are both being set to optional, with default values of `0` and `true` workable? I didn't want to make a presumption here, as I was unsure if there were cases whereby that might be surprising for the user.